challenge-02/README.md

```diff
@@ -4,29 +4,34 @@ Nesse exercício, você está livre para escolher os nomes para suas variáveis
 
 ```js
 // Crie uma função que receba dois argumentos e retorne a soma dos mesmos.
-?
+function adicao (x,y){
+	return x + y;
+}
 
 // Declare uma variável que receba a invocação da função criada acima, passando dois números quaisquer por argumento, e somando `5` ao resultado retornado da função.
-?
+? var ninja = adicao(5,10) + 5;
 
 // Qual o valor atualizado dessa variável?
-?
+20
 
 // Declare uma nova variável, sem valor.
-?
+? var ninja2
 
 /*
 Crie uma função que adicione um valor à variável criada acima, e retorne a string:
     O valor da variável agora é VALOR.
 Onde VALOR é o novo valor da variável.
 */
-?
+? > function addValor(ninja2){
```

Comment:
-1
Faltou remover o `?`.

-1
Tem um `>` perdido ali no início da linha.

-1
Essa função não precisa receber parâmetros :)